src/main/java/org/owasp/validator/html/Policy.java

```diff
@@ -120,6 +120,8 @@
      * XML Schema for policy validation
      */
     private static Schema schema = null;
+    private static boolean validateSchema = true;
+    private static SAXException savedSchemaValidationException;
```

Comment:
Seems strange to me to have this be a class variable. Can't it be passed as a parameter to the methods that need it from the caller?

Reply:
It would require an API change. All `Policy.getPolicy()` calls would need a new Boolean parameter to pass down and the method to enable/disable validation would be removed as it won’t be needed anymore.

Reply:
Sorry. I was referring to savedSchemaValidationException, not the boolean, which I think is fine. I'm testing/making changes and I'll address my concern if I easily can. Hold on for now.